Gopkg.lock

```diff
@@ -120,7 +120,7 @@
     "service/logs",
     "templates"
   ]
-  revision = "b3b2ace735601a3ec996d45fd3a48ff91fc6f849"
+  revision = "70d5cb0dd01dbd62c69476285eb2facf70f19385"
```

Comment:
Why did the vendoring change?

Reply:
Because I needed it to change :angel: :stuck_out_tongue_closed_eyes: (i.e. bumping `docker/cli` on master to get https://github.com/docker/cli/pull/1244) with `dep ensure -v -u github.com/docker/cli`. We are currently tracking `docker/cli` master in there